@here/harp-geometry/lib/SubdivisionModifier.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+    Face3,
+    Geometry,
+} from "three";
+
+/**
+ * The `SubdivisionModifier` subdivides triangle mesh geometries positioned
```

Comment:
Minor, should be [[SubdivisionModifier]]

Reply:
done